### PR TITLE
Quick runs for S3 deployment tests

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[e6a0ad2e5555de629e608a5443a63640660cb9f5] [ERROR] Errors: 
[e6a0ad2e5555de629e608a5443a63640660cb9f5] [ERROR]   JamesDeploymentValidationTest>DeploymentValidation.selectThenFetchWithExistingMessages:86 » InvalidServerResponse
[e6a0ad2e5555de629e608a5443a63640660cb9f5] [ERROR]   JamesDeploymentValidationTest>DeploymentValidation.validateDeployment:81 » InvalidServerResponse
[e6a0ad2e5555de629e608a5443a63640660cb9f5] [ERROR]   JamesDeploymentValidationTest>DeploymentValidation.validateDeploymentWithMailsFromSmtp:92 » SMTPSending
```
still happening it seems ^^'

File: src/adr/0018-hexagonal-architecture.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# 17. FileMailQueue deprecation
```
Comment (chibenwa) [inline src/adr/0018-hexagonal-architecture.md:1]: This file should be removed

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
See https://github.com/linagora/james-project/pull/3762

**Pull request closed without merging**